tx-parser/src/lib.rs

```diff
@@ -30,9 +35,19 @@ fn fn_parser<O, I>(f: fn(I) -> ParseResult<O, I>, err: &'static str) -> TxParser
     parser(f).expected(err)
 }
 
+def_parser_fn!(Tx, integer, Value, i64, input, {
```

Comment:
Should probably not have to define macros here. There's another `def_parser_fn!` as well in the mentat query parser crate. Did you intend to redefine it here? I'm having a hard time understanding what this is for as well, since it's not used anywhere.